apps/src/lib/ui/lti/sync/LtiSectionSyncDialog.tsx

```diff
@@ -120,6 +136,19 @@ export default function LtiSectionSyncDialog({
       PLATFORMS.STATSIG
     );
   };
+
+  const handleUpdateSectionOwners = () => {
+    return $.ajax({
+      url: '/lti/v1/sections/bulk_update_owners',
+      type: 'PATCH',
+      data: {
+        section_owners: sectionOwners,
+      },
+      success: () => {
```

Comment:
One thing we could do is instrument the route with a honeybadger call; otherwise any unhandled errors will still get reported to honeybadger, but possibly with less context.

Reply:
Oh I see what you mean, yeah you're right. We could potentially piggyback on the functionality we use to pass specific errors back to the front-end during sync.